src/advanced/node-management.md

```diff
@@ -755,6 +755,64 @@ equal to `start` and is less than `end`.
     That is, to collect `5` non-empty blocks from the tail of the blockchain,
     range from `288` to `6` has been traversed.
 
+## Explorer API Sockets
+
+Since Exonum 0.10 version, it is possible to connect to nodes via WebSocket.
+
+Explorer API sockets have the same base path as endpoints, denoted
+**{explorer_base_path}** and equal to `/api/explorer/v1`.
+
+Currently only one socket is implemented - it shares information on block
```

Comment:
`Currently only one socket is implemented` -> `Currently only one type of events is provided`

Reply:
done